boards/AVR_MKS_GEN_L_V1/pins_MKS_GEN_L_V1.h

```diff
@@ -181,6 +181,9 @@
 #endif
 
 //Pin to turn on dew heater MOSFET
-#ifndef DEW_HEATER_PIN
-    #define DEW_HEATER_PIN 10
+#ifndef DEW_HEATER_PIN1
+    #define DEW_HEATER_PIN1 10
 #endif
+#ifndef DEW_HEATER_PIN2
+    #define DEW_HEATER_PIN2 7
```

Comment:
please rename to DEW_HEATER_2_PIN to keep naming consistend with other definitions.

Reply:
Done